cmd/samples/recipes/ctxpropagation/README.md

```diff
@@ -0,0 +1,18 @@
+This sample workflow demos context propagation through a workflow.
+
+The workflow initializes the client with a context propagator which propagates
+across the workflow - available to use in workflow and activities.
```

Comment:
I think it requires more info why it is needed or references to specific documents that explain that.